cms/static/js/certificates/collections/signatories.js

```diff
@@ -0,0 +1,16 @@
+define([
+    'backbone', 'js/certificates/models/signatory'
+],
+function(Backbone, Signatory) {
+    'use strict';
+    console.log("signatory_collection.start");
+    var SignatoryCollection = Backbone.Collection.extend({
+        model: Signatory,
+        url: '/certificates/edX/DemoX/Demo_Course/signatories'
```

Comment:
Yes it makes sense. We should make it dynamic. Actually it is just added for the reference. Need to update in js/certificates/collections/certificates as well.